bedrock/products/views.py

```diff
@@ -104,3 +113,135 @@ def vpn_invite_waitlist(request):
+        "en-US",  # Initially, en-US is the only one available in Contentful
+    ]
+    locale = l10n_utils.get_locale(request)
+
+    template_name = "products/vpn/resource-center/landing.html"
+    ctx = {
+        "active_locales": active_locales,
+    }
+
+    resource_articles = ContentfulEntry.objects.get_entries_by_type(
+        locale=locale,
+        classification=CONTENT_CLASSIFICATION_VPN,
+        content_type=CONTENT_TYPE_PAGE_RESOURCE_CENTRE,
+    )
+    category_list = _build_category_list(resource_articles)
+    selected_category = unquote_plus(request.GET.get(ARTICLE_CATEGORY_LABEL, ""))
+
+    _filtered_articles = _filter_articles(
+        resource_articles,
+        category=selected_category,
+    )
+
+    # The resource_articles are ContentfulEntry objects at the moment, but
+    # we really only need their JSON data from here on
+    _filtered_article_data = [x.data for x in _filtered_articles]
+
+    first_article_group, second_article_group = (
+        _filtered_article_data[:ARTICLE_GROUP_SIZE],
+        _filtered_article_data[ARTICLE_GROUP_SIZE:],
+    )
+
+    ctx.update(
+        {
+            "category_list": category_list,
+            "selected_category": selected_category,
+            "first_article_group": first_article_group,
+            "second_article_group": second_article_group,
+        }
+    )
+
+    return l10n_utils.render(
+        request,
+        template_name,
+        ctx,
+        ftl_files=["products/vpn/shared", "products/vpn/resource-center"],
+    )
+
+
+def resource_center_detail_view(request, slug):
+    """Individual detail pages for the VPN Resource Center"""
+
+    template_name = "products/vpn/resource-center/article.html"
+
+    # Initially, en-US is the only one available in Contentful
+    locale = l10n_utils.get_locale(request)
+    active_locales = [
+        "en-US",
+    ]
+
+    ctx = {
+        "active_locales": active_locales,
+    }
+
+    article_dict = {}
+    try:
+        article = ContentfulEntry.objects.get_entry_by_slug(
+            slug=slug,
+            locale=locale,
+            classification=CONTENT_CLASSIFICATION_VPN,
+            content_type=CONTENT_TYPE_PAGE_RESOURCE_CENTRE,
+        )
+        article_dict.update(article.data)
+
+    except ContentfulEntry.DoesNotExist as ex:
+        capture_exception(ex)
+        # If our selected locale is valid but we get a genuine slug miss
+        # we need to 404 rather than fall through, because render()
+        # will trigger a 500 at the templating level if we don't have
+        # the article data in the context
+        if locale in active_locales:
+            raise Http404()
+
+    ctx.update(article_dict)
+
+    ctx.update(
+        {
+            "related_articles": [x.data for x in article.get_related_entries()],
+        }
+    )
+
+    return l10n_utils.render(
+        request,
+        template_name,
+        ctx,
+        ftl_files=["products/vpn/shared", "products/vpn/resource-center"],
```

Comment:
The first file in the `ftl_files` list is responsible for determining activation status for the page, so because `products/vpn/shared` is a shared file amongst many pages, we should instead put `products/vpn/resource-center` first:
```suggestion
        ftl_files=["products/vpn/resource-center", "products/vpn/shared"],
```

Reply:
good catch!

Reply:
Will these article page be localized? right now they seem to be hard-coded in English (with only English in Contentful). If that's the case, the we probably shouldn't be including `ftl` files in the view here to avoid mixed language content?

Reply:
The only parts of the pages that are localized are the voting widget and the copy of the cta below the article - but if the articles will never be translated maybe those should stay in English only?

Reply:
I guess the bigger question is: do we want to show localized pages that are 90% english, with only a small amount of localized copy? That's not something we strictly have a precedence for. Is there a long term plan to consider here for localization?

Reply:
@nathan-barrett ok, I noticed in the view that `active_locales` is being hard-coded with `en-US`, so this should be fine for avoiding mixed content.

The new FTL file added here may still end up in the L10n repo, but as long as we don't add a config for it, then it should avoid hitting Pontoon (i think?)

Reply:
These will be translated through Contentful soon, so this is preparing to support that.

Reply:
If we do want the files translated then we need to add a new config path here: https://github.com/mozilla/bedrock/blob/master/l10n/configs/pontoon.toml#L286-L297

Reply:
Awesome, thanks @alexgibson